README.md

```diff
@@ -4,121 +4,389 @@
-3. Start the WSO2 IS. 
-4. Access WSO2 IS management console and create a service provider (ex:- sampleApp)
+#### Configuring Identity Server
+1. Start the WSO2 IS. 
+2. Access WSO2 IS management console from https://localhost:9443/carbon/ and create a service provider.
+   ![Management Console](https://user-images.githubusercontent.com/15249242/91068131-6fc2d380-e651-11ea-9d0a-d58c825bbb68.png)
+   i. Navigate to the `Service Providers` tab listed under the `Identity` section in the management console and click `Add`.<br/>
+   ii. Provide a name for the Service Provider (ex:- sampleApp) and click `Register`. Now you will be redirected to the `Edit Service Provider` page.<br/>
+   iii. Expand the  `Inbound Authentication Configuration` section and click `Configure` under the `SAML2 Web SSO Configuration` section.<br/>
+   iv. Provide the following values for the respective fields and click `Update` while keeping other default settings as it is.
    
-   For the service provider, configure SAML2 Web SSO under Inbound Authentication Configuration. In this configuration,
-   use following parameters and options,
-     
        Issuer - SampleApp  
-       Assertion Consumer URLs - http://localhost:8080/SampleApp/home.jsp 
+       Assertion Consumer URLs - http://localhost.com:8080/SampleApp/home.jsp 
        Enable Attribute Profile - True
        Include Attributes in the Response Always - True
 
-
-   Keep other default settings as it is and save the configuration.
-   
-   Next, expand the [Claim Configuration](https://is.docs.wso2.com/en/latest/learn/configuring-claims-for-a-service-provider/#configuring-claims-for-a-service-provider) section. In this configuration, Set the following config and add the claims you 
+   v. Next, in the `Edit Service Provider` page, expand the 
+   [Claim Configuration](https://is.docs.wso2.com/en/latest/learn/configuring-claims-for-a-service-provider/#configuring-claims-for-a-service-provider) 
+   section. In this configuration, Set the following config and add the claims you 
    need to retrieve (ex: http://wso2.org/claims/lastname) from the web app.
    
        Select Claim mapping Dialect - Use Local Claim Dialect
        
    See the example claim config below.
    ![Claim Config](https://user-images.githubusercontent.com/15249242/90488235-38d45580-e159-11ea-8beb-52d6b5c35034.png)
+   
+   vi. Click `Update` to save.
 
-       
-5. Try out the application by accessing the `http://localhost:8080/SampleApp/index.html`.
+#### Configuring The Webapp
+1. Download the `SampleApp.war` from the [latest release](https://github.com/asgardio/asgardio-tomcat-saml-agent/releases/latest).
+2. Deploy the application, `SampleApp.war` using Apache Tomcat.
+3. Try out the application by accessing the `http://localhost.com:8080/SampleApp/index.html`.
 
-![Recordit GIF](http://g.recordit.co/IvrtWnDnZ8.gif)
+![Recordit GIF](http://g.recordit.co/IvrtWnDnZ8.gif)    
 
```

Comment:
Let's mention something about localhost.com here

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/1/commits/f25a1df8be1b219b41029eaf1195b4c5ddb16da4

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/1/commits/3135e1b3dc8a2ca466299cc3b94d6f3b8848af98